src/js/experiments/gettingStartedGuide.js

```diff
@@ -8,6 +24,7 @@ window.OptimizelyClient.getVariationName({
   guestExperiment: false,
 }).then((variation) => {
   if (variation === 'treatment') {
+    addNewBadgeToSidebar();
     // console.log('IN TREATMENT GROUP');
```

Comment:
Nit: we can remove this?

Reply:
The console log? That's part of the base branch, which I assumed we should not be changing until the end in case we are merging all the respective parts of the epic at once.